### ISSUE TITLE
Test failed in VDIF

### ISSUE THREAD

**Issue by rikvl** [mhvk/baseband#458]
After installation I get the following failed test:

```
>       assert frameset.sample_rate == 32. << u.MHz
E       assert <Quantity 32. None> == (32.0 << Unit("MHz"))
E        +  where <Quantity 32. None> = <baseband.vdif.frame.VDIFFrameSet object at 0x113711a10>.sample_rate
E        +  and   Unit("MHz") = u.MHz

baseband/vdif/tests/test_vdif.py:534: AssertionError
```

Versions used:
python: 3.7.4
numpy: 1.16.4
astropy: 4.0

**Comment by mhvk** [comment]
Very weird! I tried `tox -e test` and `pytest-3` myself, and things pass. But I have a much more recent numpy, and that could be the issue. How did you run the tests? With `tox`, in principle more up to date versions should be installed automatically.

p.s. This already found a mini-bug: in `CHANGES.rst`, I state that the minimum version of numpy required is 1.17, but I don't actually enforce it in `setup.cfg` or in the documentation, which just require `astropy>=4.0`, which in turn requires `numpy>=1.16`.

**Comment by rikvl** [comment]
The mini-bug may have been the issue. I upgraded numpy to version 1.19.2 and the test passes now (using `pytest`).

**Comment by mhvk** [comment]
OK, clearly the dependency should be explicit (the tests only cover numpy 1.17). Fix in #459
